Knossos.NET/ViewModels/Templates/DevModVersionsViewModel.cs

```diff
@@ -550,5 +583,44 @@ internal void ChangeVisibility()
                 ButtonsEnabled = true;
             }
         }
+
+        /// <summary>
+        /// DevEnv is just a version tagged "999.0.0-DevEnv" this version cant be uploaded to Nebula and only used for continuous local development process
+        /// If a Mod has this version the create devenv button is disabled as well as the upload, visibility and delete from nebula buttons
+        /// </summary>
+        internal async void CreateDevEnv()
+        {
+            if (editor == null)
+                return;
+            string devVersion = "999.0.0-DevEnv";
+            string explanation = "A DevEnv version is a mod version intended to be used for continuous local development process and it cant be uploaded to Nebula.\nThis version will always be the default active version every time you start Knet, and" +
```

Comment:
```suggestion
            string explanation = "A DevEnv version is a mod version intended to be used for continuous local development process, and it can't be uploaded to Nebula.\nThis version will always be the default active version every time you start Knet, and" +
```